internal/pkg/template/workload.go

```diff
@@ -770,6 +816,7 @@ type WorkloadOpts struct {
 	DeregistrationDelay     *int64
 	AllowedSourceIps        []string
 	NLB                     *NetworkLoadBalancer
+	ALB                     *ApplicationLoadBalancer
```

Comment:
Great callout @iamhopaul123 ❤️ 
I will remove parameters those are now part of ALB(ApplicationLoadBalancer).  `HTTPTargetContainer` we will still need because we use this config in service connect and we are touching service connect as a part of this project. So keeping it as is.